packages/api/lambdas/create-reconciliation-report.js

```diff
@@ -900,9 +895,7 @@ async function processRequest(params) {
     location: buildS3Uri(systemBucket, reportKey),
   };
   let [reportPgRecord] = await reconciliationReportPgModel.create(knex, builtReportRecord);
-  let reportApiRecord = translatePostgresReconReportToApiReconReport(reportPgRecord);
-  await indexReconciliationReport(esClient, reportApiRecord, process.env.ES_INDEX);
-  log.info(`Report added to database as pending: ${JSON.stringify(reportApiRecord)}.`);
+  log.info(`Report added to database as pending: ${JSON.stringify(reportPgRecord)}.`);
```

Comment:
Is there no value in logging the API-formatted record as we did previously? Generally users don't see the pg record, they get the translated version. But if we're just translating to log it... that may be overkill.

Reply:
So yeah, the way this is written, we are translating it at the very end to return the api format record, but this seems to be the only place there was explicit logging. I'm not sure for logging purposes if api format is better than the pg format, so I'm open to doing it that way. But I did just log what we had instead of translating it for logging.
Also, thinking about it more, I'm open to adding some more logging like maybe at the end before returning the api format record. What do you think?

Reply:
If it doesn't make sense to move this log statement or translated beforehand, maybe just update the message to make it clear that it's a PG-formatted record. It's a nit. I'm just concerned that someone sees this in the logs and gets confused as to why the naming conventions are different. More logging is good.

Reply:
for consistency, it seems correct to keep the translation and logging of the api format. the translation is a quick function anyways. Added a comment for future reference because it might be confusing why the translation was done just for logging.
Also added a bit more logging (with translation)